src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/SupplementaryEvidenceTest.java

```diff
@@ -60,6 +61,7 @@ public void setup() {
         );
     }
 
+    @DisplayName("Should attach supplementary evidence documents to the existing case")
     @Test
     public void should_attach_supplementary_evidence_to_the_case() throws Exception {
```

Comment:
I wonder if this brings any value if the test names are as descriptive as they are right now. Especially when underscores are used.

IMHO duplicating  test name in the annotation adds a risk of them to go out of sync. And it's just more code to maintain.

Reply:
removing the description